src/names/common.h

```diff
@@ -485,4 +485,16 @@ class CNameCache
 
 };
 
+struct NameNewReturn
+{
+    bool ok;
+    std::string err_msg;
+    std::string toaddress;
+    std::string hex;
+    std::string rand;
+    std::string data;
```

Comment:
The reason I have them as strings is because the only use for this information is to turn it into a UniValue JSON object and send to RPC name_firstupdate, which needs to be strings. I can change it if it's important.